examples/misc_exporter_collada.html

```diff
@@ -352,7 +352,7 @@
 
 			function exportCollada() {
 
-				const result = exporter.parse( teapot );
+				const result = exporter.parse( teapot, undefined, { upAxis: 'Y_UP', unitName: 'millimeter', unitMeter: 0.001 } );
```

Comment:
May I ask how you come up with this unit? Isn't it more common to produce `<unit name="meter" meter="1"/>`?

Reply:
I kept the scale of the scene closer to what it was originally, it also demonstrates better if the scale factor is actually being applied.